### PR TITLE
[1LP][RFR] NewTest:: testing network configuration using negative input

### PR DESCRIPTION
**Pull request by dgaikwad**
### PRT Run
- Testing network configuration using negative input
Testing below options:
1) Set DHCP Network Configuration
4) Test Network Configuration
5) Set Hostname

{{pytest: cfme/tests/cli/test_appliance_console.py::test_appliance_console_network_conf_negative --long-running -v}}

### CODE REVIEW

File: cfme/tests/cli/test_appliance_console.py [hunk 1]
```diff
@@ -1007,7 +1011,42 @@ def test_appliance_console_network_conf_negative():
...
+
+    new_config_file = fauxfactory.gen_alphanumeric(start="test_", length=10)
+    command_set = (f"ap | tee -a /tmp/{new_config_file}", RETURN)
+    appliance.appliance_console.run_commands(command_set, timeout=30)
+    command_set = (f"cat /tmp/{new_config_file}")
+    new_data = appliance.ssh_client.run_command(command_set)
+    assert new_data.success
+    new_data = new_data.output.split("\n")
+    logger.info("\'ap\' command's output after test:{}".format(new_data))
+
```
Comment (ganeshhubale) [inline cfme/tests/cli/test_appliance_console.py]: same here for fstring

File: cfme/tests/cli/test_appliance_console.py [hunk 2]
```diff
@@ -994,20 +1022,59 @@ def test_appliance_console_network_conf_negative():
...
+        assert "Please provide a valid Hostname or IP Address." in result[-1], (
+            "Not getting error message for invalid error for IPV6")
+
+        command_set = ("ap", RETURN, "1", "5", invalid_hostname)
+        appliance.appliance_console.run_commands(command_set, timeout=30)
+
+        new_config_file = fauxfactory.gen_alphanumeric(start="test_", length=10)
+        command_set = (f"ap | tee -a /tmp/{new_config_file}", RETURN)
+        appliance.appliance_console.run_commands(command_set, timeout=30)
+        command_set = (f"cat /tmp/{new_config_file}")
```
Comment (digitronik) [inline cfme/tests/cli/test_appliance_console.py]: Why we need to create extra file with data and then cat it?
`run_commands` output arg not helpful here?

**Comment by dgaikwad** [comment]
Following action performed after cove review:
- Removed parseIPV6() and parseIPV4() functions, this function will be implemented as you suggested and will use same where we need.
- Improved formatting
- updated logic where I was storing "ap" command's output in the text file

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**